sim/python/vernier.py

```diff
@@ -243,9 +250,9 @@ def plot_vernier_pll(device, refin, fsamp=125e6, jitter=1e-10, npts=8000000, sho
     fref = freq_vernier(device, refin, 20e6)
     # Simulate the PLL with a moderate phase step.
     t = np.arange(npts) / fsamp
-    (_, step_phase, _, _) = vernier_pll(npts, fsamp, fref, 0.0, 5.0e-9, jitter)
+    (_, step_phase, _, _, _) = vernier_pll(npts, fsamp, fref, 0.0, 5.0e-9, jitter)
```

Comment:
This could be `_, step_phase, *_ = vernier_pll(npts, fsamp, fref, 0.0, 5e-9, jitter)` but it only makes sense if you have an absurd number of returns.